### PR TITLE
Community mem

### LINKED ISSUE #4126: [Community] Member Profile: Naureen
**Current Behavior**

@goldfishdolphin has been a consistent contributor and community member.

**Desired Situation**
Let's recognize @goldfishdolphin as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: @goldfishdolphin
- Twitter: 
- LinkedIn: [naureen-imran-744932227](https://www.linkedin.com/in/naureen-imran-744932227/)
- Link to profile picture:

A detailed explanation on how to set up a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by Saksham294**
**Description**

This PR fixes #4126 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by Saksham294** [comment]
I don't know why a commit from another branch is coming here. In my terminal also only 1 was changed which was specifically for this issue only.

**Comment by l5io** [comment]
🚀 Preview for commit 476abae426a009eb1aa41380d526e42c92c2d29c at: https://644a7c96e45f39008e5ac676--layer5.netlify.app

File: src/sections/Community/community.style.js
Comment (vishalvivekm) [inline src/sections/Community/community.style.js:1]: 1. Keep your local repo updated with the upstream
2. before starting making changes for a particular issue create a branch off of the ` main ` branch 
3. you can avoid using ` git add . ` to some extent when you aren't sure what all files might be included for staging.

for now, you can try interactive rebase to only pick the commit that's required for the issue.

**Comment by leecalcote** [comment]
@goldfishdolphin to review

**Comment by goldfishdolphin** [comment]
Thank you @Saksham294 for adding me as a community member. It is looking great.
Can you please change my location to Manchester instead of Manchester Area?
I would suggest for the future making your commit messages in the present tense as a convention. For example “Add community member”

To deal with unwanted commit you can follow @vishalvivekm’s suggestions.

**Comment by Saksham294** [comment]
I am really sorry for making mistakes in this one, I actually did not know how a commit from other branch become a part of this branch. I have done the interactive rebase, check if it is fine. Thanks a lot to all of you.

**Comment by l5io** [comment]
🚀 Preview for commit 57b8c17b8eb01f4390724d669fb6dc5ee0da2c27 at: https://644bd2b044ccd1195880ff6d--layer5.netlify.app

**Comment by vishalvivekm** [comment]
LGTM
Thanks @Saksham294

**Review by Chadha93 — APPROVED** [review]
LGTM

**Pull request merged**